test/test_visualization.jl

```diff
@@ -263,6 +263,13 @@ isdir(outdir) && rm(outdir, recursive=true)
       end
     end
   end
+
+  @testset "Makie visualization tests for UnstructuredMesh2D" begin
+    @test_nowarn_debug trixi_include(@__MODULE__, joinpath(examples_dir(), "unstructured_2d_dgsem", "elixir_euler_wall_bc.jl"))
+    @test_nowarn_debug Makie.plot(sol) # test heatmap plot
+    @test_nowarn_debug Trixi.trixi_plot(sol) # test interactive surface plot
```

Comment:
Currently yes. It will work for `TreeMesh` and `StructuredMesh`, but is restricted to `UnstructuredMesh2D` for now. 

I want to extend to other mesh types, but I was hoping to do so after I add Plots.jl support in #755 so that I don't break any existing plotting functionality accidentally.

Reply:
:+1: